### ISSUE TITLE
Add embed option to show an alert to AdBlock users

### ISSUE THREAD

**Issue by FlorianWetschoreck** [vega/vega-embed#109]
Users who use an AdBlocker might not see charts which load external datasets.
See the following issue in altair:
https://github.com/altair-viz/altair/issues/1199

This is also an issue across a wide range of blog posts which use vega-embed.

The proposed solution is to add an embed option like "adblock_alert".
If the flag is set to true, vega-embed should check if external datasets can be loaded. If this is not the case, an alert should be shown to the user that he should consider deactivating the AdBlocker if the chart is broken.

Looking forward to your thoughts about this :)

Also, I am available to implement this if we agree on a solution..

**Comment by domoritz** [comment]
I use uBlock origin and I don't see this problem. First,let's file an issue with the adblocker since it is clearly making a mistake here, no?

**Comment by domoritz** [comment]
I don't see why we a) need an option to enable this (shouldn't this be the default) b) how embed would be able to tell since embed doesn't intercept the loader.

**Comment by FlorianWetschoreck** [comment]
Thank you for your fast response :)

And yes, maybe everything will be solved when the AdBlocker is correcting its behavior. So far, I have only seen embeds fail which included vega-datasets, so maybe the AdBlocker is confusing this URL.

The idea for solving this was that vega-embed would try to load external data itself and check if it works. If it fails, it would show the alert.
For this, vega-embed either also needs to know the external URL or just have a flag. In situations, when no external datasets are loaded, you might not want to do the check. Hence the idea with the flag.

**Comment by domoritz** [comment]
Thanks for flagging this. 

I think the first solution is to not rely on the data from vega.github.io and instead serve the data from the Altair website. I installed the adblocker and can reproduce the issue but it's really nothing I think we can/should fix here.

**Comment by domoritz** [comment]
The problem is 

<img width="459" alt="screen shot 2018-11-21 at 13 51 08" src="https://user-images.githubusercontent.com/589034/48870149-8826ff80-ed94-11e8-9a4e-bf7beb993603.png">

**Comment by domoritz** [comment]
The list is at https://raw.githubusercontent.com/hoshsadiq/adblock-nocoin-list/master/nocoin.txt

**Comment by domoritz** [comment]
The real issue is https://github.com/hoshsadiq/adblock-nocoin-list/issues/332

**Comment by FlorianWetschoreck** [comment]
Thank you for finding this issue so quickly :)

**Comment by FlorianWetschoreck** [comment]
Just out of curiosity: how did you find this issue so quickly?

**Comment by domoritz** [comment]
Experience with debugging my own code ;-)

I installed the adblocker, the saw that GitHub domains are blocked (browser dev tools). Then I check which filter list in the adblocker causes the problem (trial and error). I noticed it's the bitcoin list. I then googled for that list and also noticed that the list is linked to from the tooltip in the adblocker's settings. I found the offending line and when I wanted to create an issue, I noticed that somebody else already created an issue.
